content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -78,6 +78,16 @@ that case, the documentation for the signer should specify the meaning of
 additional certificates; for example, this might be the certificate plus
 intermediates to be presented during TLS handshakes.
 
+The PKCS#10 signing request format doesn't allow to specify a certificate
+expiration or lifetime. The expiration or lifetime therefore has to be set
```

Comment:
nit: The PKCS#10 signing request format doesn't allow you to specify a certificate expiration or lifetime.
nit: Replace "e.g." with ",for example,"